Modules/IO/GDCM/src/itkGDCMImageIO.cxx

```diff
@@ -1444,6 +1444,24 @@ bool GDCMImageIO::GetLabelFromTag(const std::string & tag,
   return false;
 }
 
+void GDCMImageIO::InternalSetCompressor(const std::string &_compressor )
+{
+
+  if (_compressor == "" ||
```

Comment:
As said above this is missing other important compressor (RLE & JPEG-LS)

Reply:
future work.